mission-control/docs/installation/_gke_iam.mdx

```diff
@@ -0,0 +1,112 @@
+import Domain from '@site/docs/partials/_domain.mdx'
+
+## Create an IAM Role
+
+Depending on how you want to use Mission Control you need to create an IAM role for mission control to use:
+
+| Use Case                                     | Role           |
+| -------------------------------------------- | ---------------|
+| Read Only Scraping                           | `roles/viewer` |
+| Playbooks to create and update GCP Resources | `roles/editor` |
+
+## Configure IAM Roles for Mission Control
+
+<Tabs>
+  <TabItem label="Workload Identity" value="Workload Identity">
+
+    <Tabs>
+      <TabItem label="Bind policy to service account" value="federation">
+        
+        You can also refer the official docs: https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
+
+        1. Enable workload identity
+            ```bash
+            # The name of the GKE cluster mission control is being deployed to
+            export CLUSTER=<CLUSTER_NAME>
+            # the default namespace the mission-control helm chart uses
+            export NAMESPACE=mission-control
+            # GCP Project ID
+            export PROJECT_ID=gcp-project-id
+            # Location of GKE Cluster
+            LOCATION=us-east1
+
+            gcloud container clusters update $CLUSTER \
+                --location=$LOCATION \
+                --workload-pool=PROJECT_ID.svc.id.goog
+            ```
+            <p/>
+
+        2. Bind IAM Policy
+
+           The `$KSA_NAME` refers to the Kubernetes service account name. In our case, we need to bind to 3 service accounts: `mission-control`, `canary-checker` and `config-db`
+
+            ```bash
+            gcloud projects add-iam-policy-binding projects/$PROJECT_ID \
```

Comment:
This should be copy and pastable, I think the service accounts are also all suffixed with `-sa`